nautobot/core/settings.json

```diff
@@ -0,0 +1,598 @@
+                        },
+                        "TIMEOUT": {
+                            "type": "integer",
+                            "default": 300
+                        },
+                        "OPTIONS": {
+                            "type": "object",
+                            "properties": {
+                                "CLIENT_CLASS": {
+                                    "type": "string",
+                                    "default": "django_redis.client.DefaultClient"
+                                },
+                                "PASSWORD": {
+                                    "type": "string",
+                                    "default": ""
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "CELERY_BROKER_TRANSPORT_OPTIONS": {
+            "type": "object",
+            "default": {}
+        },
+        "CELERY_BROKER_URL": {
+            "type": "string",
+            "default": "redis://localhost:6379/0"
+        },
+        "CELERY_TASK_DEFAULT_QUEUE": {
+            "type": "string",
+            "default": "default"
+        },
+        "CELERY_TASK_SOFT_TIME_LIMIT": {
+            "type": "integer",
+            "default": 300
+        },
+        "CELERY_TASK_TIME_LIMIT": {
+            "type": "integer",
+            "default": 600
+        },
+        "CELERY_WORKER_PROMETHEUS_PORTS": {
+            "type": "array",
+            "items": {
+                "type": "integer"
+            },
+            "default": []
+        },
+        "CELERY_WORKER_REDIRECT_STDOUTS": {
+            "type": "boolean",
+            "default": true
+        },
+        "CELERY_WORKER_REDIRECT_STDOUTS_LEVEL": {
+            "type": "string",
+            "default": "WARNING"
+        },
+        "CHANGELOG_RETENTION": {
+            "type": "integer",
+            "default": 90
+        },
+        "CONFIG_CONTEXT_DYNAMIC_GROUPS_ENALBED": {
+            "type": "boolean",
+            "default": false
+        },
+        "CONTENT_TYPE_CACHE_TIMEOUT": {
+            "type": "integer",
+            "default": 0
+        },
+        "CORS_ALLOW_ALL_ORIGINS": {
+            "type": "boolean",
+            "default": false
+        },
+        "CORS_ALLOWED_ORIGINS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "CORS_ALLOWED_ORIGIN_REGEXES": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/regex"
+            },
+            "default": []
+        },
+        "CSRF_TRUSTED_ORIGINS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "DATABASES": {
+            "type": "object",
+            "properties": {
+                "default": {
```

Comment:
Possible to have additional databases beyond `default`, so be sure to have `additionalProperties: true` on this one.

More generally, it'd probably be good to make sure we permit the additional configs for the version-control app (https://github.com/nautobot/nautobot-app-version-control/?tab=readme-ov-file#configuring-nautobot-to-use-version-control).

Reply:
https://github.com/nautobot/nautobot/pull/5235/commits/35537e938779da6d2316a811d4986f6d927487d5